scripts/package/create_packages.sh

```diff
@@ -16,6 +16,7 @@ find $DEST_PACKAGES \( -name '*.nupkg' -o -name '*.snupkg' \) -exec rm {} \;
 # Setup package metadata
 REVISION=$(git rev-parse HEAD)
 BRANCH=dev
+dotnet restore
```

Comment:
make sure to pass the solution | project

Reply:
done! Thanks for catching that.